Formula/poppler.rb

```diff
@@ -32,12 +33,12 @@ class Poppler < Formula
   depends_on "little-cms2"
   depends_on "nss"
   depends_on "openjpeg"
-  depends_on "qt@5"
+  depends_on "qt"
 
   uses_from_macos "gperf" => :build
   uses_from_macos "curl"
 
-  conflicts_with "pdftohtml", "pdf2image", "xpdf",
+  conflicts_with "pdftohtml", "pdf2image", "xpdf", "poppler-qt5",
```

Comment:
```suggestion
  conflicts_with "pdftohtml", "pdf2image", "xpdf",
```
You can remove this now that `poppler-qt5` is keg-only.